packages/ember-htmlbars/tests/integration/component_invocation_test.js

```diff
@@ -53,516 +34,6 @@ QUnit.module('component - invocation', {
-  }).create();
-
-  runAppend(view);
-
-  equal(jQuery('#qunit-fixture').text(), 'In template');
-});
-
-QUnit.test('hasBlock is false when no block supplied', function() {
-  expect(1);
-
-  owner.register('template:components/with-block', compile('{{#if hasBlock}}{{yield}}{{else}}No Block!{{/if}}'));
-
-  view = EmberView.extend({
-    [OWNER]: owner,
-    template: compile('{{with-block}}')
-  }).create();
-
-  runAppend(view);
-
-  equal(jQuery('#qunit-fixture').text(), 'No Block!');
-});
-
-QUnit.test('hasBlockParams is true when block param supplied', function() {
-  expect(1);
-
-  owner.register('template:components/with-block', compile('{{#if hasBlockParams}}{{yield this}} - In Component{{else}}{{yield}} No Block!{{/if}}'));
-
-  view = EmberView.extend({
-    [OWNER]: owner,
-    template: compile('{{#with-block as |something|}}In template{{/with-block}}')
-  }).create();
-
-  runAppend(view);
-
-  equal(jQuery('#qunit-fixture').text(), 'In template - In Component');
-});
-
-QUnit.test('hasBlockParams is false when no block param supplied', function() {
-  expect(1);
-
-  owner.register('template:components/with-block', compile('{{#if hasBlockParams}}{{yield this}}{{else}}{{yield}} No Block Param!{{/if}}'));
-
-  view = EmberView.extend({
-    [OWNER]: owner,
-    template: compile('{{#with-block}}In block{{/with-block}}')
-  }).create();
-
-  runAppend(view);
-
-  equal(jQuery('#qunit-fixture').text(), 'In block No Block Param!');
-});
-
-QUnit.test('static named positional parameters', function() {
-  var SampleComponent = Component.extend();
-  SampleComponent.reopenClass({
-    positionalParams: ['name', 'age']
-  });
-  owner.register('template:components/sample-component', compile('{{attrs.name}}{{attrs.age}}'));
-  owner.register('component:sample-component', SampleComponent);
-
-  view = EmberView.extend({
-    [OWNER]: owner,
-    layout: compile('{{sample-component "Quint" 4}}')
-  }).create();
-
-  runAppend(view);
-
-  equal(jQuery('#qunit-fixture').text(), 'Quint4');
-});
-
-QUnit.test('dynamic named positional parameters', function() {
-  var SampleComponent = Component.extend();
-  SampleComponent.reopenClass({
-    positionalParams: ['name', 'age']
-  });
-
-  owner.register('template:components/sample-component', compile('{{attrs.name}}{{attrs.age}}'));
-  owner.register('component:sample-component', SampleComponent);
-
-  view = EmberView.extend({
-    [OWNER]: owner,
-    layout: compile('{{sample-component myName myAge}}'),
-    context: {
-      myName: 'Quint',
-      myAge: 4
-    }
-  }).create();
-
-  runAppend(view);
-
-  equal(jQuery('#qunit-fixture').text(), 'Quint4');
-  run(function() {
-    set(view.context, 'myName', 'Edward');
-    set(view.context, 'myAge', '5');
-  });
-
-  equal(jQuery('#qunit-fixture').text(), 'Edward5');
-});
-
-QUnit.test('if a value is passed as a non-positional parameter, it takes precedence over the named one', function() {
```

Comment:
:heavy_check_mark: if a value is passed as a non-positional parameter, it raises an assertion

@Serabe thank you so much for noticing and fixing the test names :smile: :smile: :smile: